shared-libs/lineage/src/hydration.js

```diff
@@ -218,6 +218,7 @@ module.exports = function(Promise, DB) {
           })
           .then(function(contacts) {
             fillContactsInDocs(lineage, contacts);
+            fillContactsInDocs(patientLineage, contacts);
```

Comment:
Fixes this bug: https://github.com/medic/cht-core/pull/6325#issuecomment-617613788

Reply:
Nice!